qemu/tests/pci_bridge_block_stress.py

```diff
@@ -0,0 +1,66 @@
+import logging
+
+from autotest.client.shared import error
+from virttest import utils_test
+from virttest import env_process
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    [pci-bridge]Check stress when 31 block devices attached to 1 pci-bridge, this case will:
```

Comment:
Set block device number to 10 temporarily for blocking by Bug 1410315 - [KVM_AUTOTEST] RFE: Failed to insert more than 19 block devices under pci-bridge.